### ISSUE TITLE
Simultaneous Vertical and Horizontal Scroll

### ISSUE THREAD

**Issue by JoshHiggins** [alvarotrigo/fullPage.js#2156]
When clinking an anchor for a slide in another section, the page scrolls vertically to the section and then slides horizontal to the requested slide. 

Is it possible to have the vertical and horizontal slides scroll at the same time?

**Comment by alvarotrigo** [comment]
> Is it possible to have the vertical and horizontal slides scroll at the same time?

Nop. And I believe it would look quite weird :)

**Comment by JoshHiggins** [comment]
They wouldn't need to scroll at the same speed, just to initiate scroll at the same time. I got some feedback during testing a new site that users we're finding it bothersome to have the page scroll to content they didn't request, pause and then scroll again to the content they did request.

**Comment by alvarotrigo** [comment]
I'll mark it as an enhancement in case I think about doing it in a future. 
If you really need it now [contact me](http://alvarotrigo.com/#contact-page) for a customization.

**Comment by etlingfs** [comment]
+1 for something like this. Rather than having a simultaneous vertical and horizontal scroll, I envision it being more of a "hidden slide scroll." Specifically I would like to be on say `Section 2, Slide 1` and be able to scroll up and land on `Section 1, Slide 5`, but only see a vertical scrolling motion (the default section up/down animation) rather than having to see `Section 1, Slide 1` and then a transition from `Slide 1 to Slide 5.`

Basically it would create the option to continuously scroll a story, while maintaining the narrative path/flow between sections.

@JoshHiggins is that at all similar to what you are describing? Or are we after different things.

**Comment by alvarotrigo** [comment]
> Basically it would create the option to continuously scroll a story, while maintaining the narrative path/flow between sections.

Have you checked the [scroll horizontally](http://alvarotrigo.com/fullPage/extensions/scroll-horizontally.html) extension?

**Comment by etlingfs** [comment]
@alvarotrigo - yes! I am using it currently. It is perfect, except for the above scenario, where I am using multiple sections, with slides in each section, and I want to be able to scroll from Section B, Slide 1, back up to to Section A, Slide 5, without a bunch of extra animations. Let me know if there is a better way of properly describing my desired behavior, I think I'm having a hard time putting it into words.

**Comment by alvarotrigo** [comment]
@etlingfs please open a new issue and explain step by step how to reproduce your issue. 
I can't. Section 2A - scroll up- goes to Section 1.C

**Comment by etlingfs** [comment]
@alvarotrigo - OK, will do, thanks. I will upload an example as well.

**Comment by etlingfs** [comment]
@alvarotrigo - description of desired behavior is here: https://github.com/alvarotrigo/fullPage.js/issues/2499
